client/components/state-dropdown/index.js

```diff
@@ -27,7 +27,6 @@ const StateDropdown = ( props ) => {
 StateDropdown.propTypes = {
 	id: PropTypes.string.isRequired,
 	countriesData: PropTypes.object.isRequired,
-	valuesMap: PropTypes.object.isRequired,
```

Comment:
Same as above, the `StateDropdown` component generates its own `valuesMap` from the `countriesData` object.